README.md

```
@@ -121,9 +123,30 @@ const options: StampOptions = {
+1. **Comment**: Injects a comment at the end of the `body` tag (by default).
+2. **Meta Tag**: Injects a meta tag in the `head` section of the HTML.
+
+You can choose the mode by setting the `mode` option in the `StampOptions`.
+
+### 🦋 Options
+
+|      Field       |                     Type                     |                                             Description                                              |                           Default                           |
+|:----------------:|:--------------------------------------------:|:----------------------------------------------------------------------------------------------------:|:-----------------------------------------------------------:|
+| `targetSelector` |                   `string`                   | CSS selector to find the target element to inject the stamp. Only used if `mode` is set to `comment` |                          `"body"`                           |
+|      `mode`      |          `"comment" \| "meta-tag"`           |                           Mode of HTML injection: `comment` or `meta-tag`                            |                         `"comment"`                         |
+|    `metaTag`     |  [StampMetaTagOptions](#-meta-tag-options)   |        Options related to the `meta-tag` injection. Only used if `mode` is set to `meta-tag`         | Refer to [Meta Tag Options](#-meta-tag-options) for details |
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)